tasks/beta-releases.js

```diff
@@ -96,6 +103,7 @@ function getChangelog() {
   const commitsArray = getCommits();
 
   commitsArray.forEach(commit => {
+    commit = commit.replace(delimeter, '');
```

Comment:
Wondering if we still need this line, given we split the original string by the delimiter. I may be missing something though, and having this like for the safety sake sounds good tome.

Reply:
Hi @asudoh ,

I added it because when the changelog was generated, the last item for some reason still had the delimiter text in there. I ended up manually removing it from the test beta release (https://github.com/carbon-design-system/ibm-dotcom-library/releases/tag/1.7.1-beta.4303)

Reply:
I see, makes sense, thank you for clarifying @jeffchew!